src/lib/controller.js

```diff
@@ -0,0 +1,79 @@
+import {store} from "./store";
+import {updateUrl} from "./fb";
+import {runLighthouse, fetchReports} from "./ci";
+
+class Controller {
```

Comment:
Instead of creating a `Controller` class, what do you think of turning these into Redux/Unistore actions?

https://github.com/developit/unistore/blob/master/README.md#action

I think that would go something like:

```
export const requestRunLighthouse = store.action(async (state) => {
  .. do a bunch of async stuff ...
  return {
    userUrl: url,
    ...
  };
});
```

Maybe we'd create an actions directory and have one action per file?

Reply:
FWIW there's no actions-in-serial behavior here. I should have expected that from a @developit 350 byte special ;-)

Reply:
Done, take a look.